File: C++/AR_460.cpp
#include <iostream>
#include <string>
#include <cmath>

using namespace std;

string baseDiez(string n) {		

	string ret = "";
	
	for (int i = 0; i < n.size(); ++i) {
		ret += n[i];
		if(i != n.size() -1 ) ret += "0";
	}

	/*--i;
	while (i >= 0) {
		string auxS = n[i] + "0";
		ret = auxS + ret;

		i--;
	}*/


	return ret;
}

bool resuelveCaso() {
	string n = "";
	cin >> n;
	if (!cin)
		return false;
	
	cout << baseDiez(n) << "\n";

	return true;
}

int main() {

	while (resuelveCaso()) {}

	return 0;
}

File: Java/AR_460.java
import java.io.*;

public class AR_460 {
    
    public static String baseDiez(String n) {        
        StringBuilder ret = new StringBuilder();
        
        for (int i = 0; i < n.length(); ++i) {
            ret.append(n.charAt(i));
            if (i != n.length() - 1) {
                ret.append("0");
            }
        }
        return ret.toString();
    }

    public static boolean resuelveCaso(BufferedReader br, BufferedWriter bw) throws IOException {
        String n = br.readLine();
        if (n == null || n.isEmpty()) return false;
        
        bw.write(baseDiez(n) + "\n");
        bw.flush();
        return true;
    }

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        
        while (resuelveCaso(reader, bw));

        br.close();
        bw.close();
    }
}
